app/cmd/plugin_search.go

```diff
@@ -71,19 +71,69 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
-// Output output the data into buffer
+func showVersionPlugins(plugins []client.AvailablePlugin) (p []client.CenterPlugin) {
+	p = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	if err != nil {
+		log.Fatal(err)
+	}
+	jclient1 := &client.PluginManager{
```

Comment:
`jclient1` is a bad naming practice. Plus, we already have it at here https://github.com/jenkins-zh/jenkins-cli/pull/132/files#diff-a77a67bf776e10119612072fdd491d2fR40.